runtime/gc_stats/ScavengerJavaStats.cpp

```diff
@@ -65,4 +73,8 @@ MM_ScavengerJavaStats::mergeOwnableSynchronizerCounts(MM_ScavengerJavaStats *sta
 	_ownableSynchronizerCandidates += statsToMerge->_ownableSynchronizerCandidates;
 	_ownableSynchronizerTotalSurvived += statsToMerge->_ownableSynchronizerTotalSurvived;
 	_ownableSynchronizerNurserySurvived += statsToMerge->_ownableSynchronizerNurserySurvived;
+
+	_monitorReferenceCleared += statsToMerge->_monitorReferenceCleared;
+	_monitorReferenceCandidates += statsToMerge->_monitorReferenceCandidates;
+
```

Comment:
No need for new line here

Reply:
monitor ref merge should not be in this method

Reply:
Is it being merged elsewhere?

Reply:
Looks like you're doing it in `MM_ScavengerDelegate::mergeGCStats_mergeLangStats`, which should be sufficient